src/components/ZclAttributeManager.vue

```diff
@@ -276,13 +279,40 @@ export default {
         return false
       }
     },
-    isDisabled(id, name, selectedClusterId, selectedCluster) {
+    //disabling default field if Storage is External AND if attribute is not enabled
+    isDisabledDefault(id, name, selectedClusterId) {
+      return (
+        !this.selection.includes(
+          this.hashAttributeIdClusterId(id, selectedClusterId)
+        ) ||
+        this.selectionStorageOption[
+          this.hashAttributeIdClusterId(id, selectedClusterId)
+        ] == 'External'
+      )
+    },
+    //disabling Storage if forced External AND if attribute is not enabled
```

Comment:
comment=> Returns true if storage is forced as External and attribute is disabled. ?

Reply:
good point, ill add Returns true
I kinda thought it was implied